docker-compose.yaml

```diff
@@ -72,6 +72,10 @@ services:
     ports:
       - 8200:8200
       - 8201:8201
+    extra_hosts:
+        # Makes this name resolve inside the container for Linux test hosts (used for Statsd Telemetry addresss)
+        # https://docs.docker.com/desktop/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host
+      - host.docker.internal:host-gateway
```

Comment:
This is a static mapping to resolve the IP of the host inside the containers. The name will be used again in the Statsd telemetry stanza below.